src/utils.rs

```diff
@@ -132,6 +135,60 @@ where
         .expect("failed to create monitor thread")
 }
 
+// Temporary functions to convert ValueDeletable to Option<Bytes> until
+// we add proper support for merges.
+pub(crate) fn unwrap_result(value: ValueDeletable) -> Result<Option<Bytes>, SlateDBError> {
+    match value {
+        ValueDeletable::Value(v) => Ok(Some(v)),
+        ValueDeletable::Merge(_) => {
+            unimplemented!("MergeOperator is not yet fully implemented")
+        }
+        ValueDeletable::Tombstone => Ok(None),
+    }
+}
+
+pub(crate) async fn get_now_for_ttl(
```

Comment:
thoughts on `now_for_read`? all reads should probably use this method

Reply:
fair enough